workspace_host/interface.js

```diff
@@ -545,15 +376,6 @@ async function dockerAttemptKillAndRemove(container) {
     Sentry.captureException(err);
     logger.error('Error removing stopped container', err);
   }
-
-  if (name) {
-    const workspaceJobPath = path.join(workspacePrefix, name);
-    try {
-      await fsPromises.rmdir(workspaceJobPath, { recursive: true });
-    } catch (err) {
-      logger.error(`Error removing directory ${workspaceJobPath}`, err);
-    }
-  }
```

Comment:
This was broken code to begin with; we've been seeing errors like the following in prod for quite some time now:

```
Error removing directory /jobs/workspace-a80b4e08-a522-45d5-9d7b-03cc51207ac9 ENOENT: no such file or directory, stat '/jobs/workspace-a80b4e08-a522-45d5-9d7...'
```